src/views/Home/components/DiningBalance/index.jsx

```diff
@@ -4,8 +4,8 @@ import { useEffect, useState } from 'react';
 import { Doughnut } from 'react-chartjs-2';
 import session from 'services/session';
 import user from 'services/user';
-import { gordonColors } from 'theme';
 import styles from '../Doughnut.module.css';
+import { newTheme } from 'theme';
```

Comment:
Consider "theme360" instead of "newTheme" everywhere, since it won't always be new.